pkg/command/buildpacks.go

```diff
@@ -156,7 +158,12 @@ func (opts *BuildpackOptions) Execute(cmd *cobra.Command, args []string) error {
 }
 
 func (opts *BuildpackOptions) build(ctx context.Context, sourceDigest name.Digest, w io.Writer) (name.Digest, error) {
-	tag, err := opts.tag()
+	tag, err := opts.tag(imageNameContext{
+		URL: url.URL{
+			Scheme: "buildpack",
```

Comment:
```suggestion
			Scheme: "buildpacks",
```

(Unless there's a bug elsewhere? Seems like these need to be the same.)

IDK if you control the scheme, but it kills me that `buildpacks` is plural and `dockerfile` is singular, but maybe it makes sense to you.

Reply:
`buildpack` is correct (in the sense that it's consistent with what resolve/apply accept): https://github.com/mattmoor/mink/blob/188bc51883d9eaf0ce0bc42374b8c3ab632a40a8/pkg/command/resolve.go#L146

Reply:
ok, I see you found a bug in the crazy template examples 👍